elcoq.el

```diff
@@ -476,18 +481,19 @@ sertop running."
     (let ((str (buffer-substring-no-properties
                 (overlay-start overlay)
                 (overlay-end overlay))))
-      `(Control (StmAdd
-                 1 ;; Just one sentence for now
-                 ,(if previous-state `(Some ,previous-state) `None)
-                 ,str)))))
+      ;; If no options given, StmAdd will try "the rigth thing"
+      `(Control (StmAdd () ,str)))))
+                 ;; 1 ;; Just one sentence for now
+                 ;; ,(if previous-state `(Some ,previous-state) `None)
+                 ;; ,str)))))
```

Comment:
This doesn't look good: elcoq doesn't keep track of what state is current, so we'd much rather have an add on the wrond state id rejected than have it silently accepted and added somewhere else in the document, right?